src/mine-sweeper.tsx

```diff
@@ -98,8 +98,8 @@ export default class MineSweeper extends React.Component<IMineSweeperProps, IMin
   }
 
   public render() {
-    const grid = this.state.mineField.map((row: any, i: any) => {
-      const cells = row.map((_: any, j: any) => {
```

Comment:
I removed the `any` types here, because TypeScript can actually infer the types on a map function. Putting `any` on them will make `i` and `j` lose their `number`- type.